.github/workflows/ci.yaml

```diff
@@ -78,18 +78,21 @@ jobs:
     if: ${{needs.changed_files.outputs.rs_changed == 'true'}}
     steps:
       - uses: actions/checkout@v3
+      - uses: arduino/setup-protoc@v1
       # Otherwise the arguments to the setup-rust-toolchain action are ignored.
       - run: rm rust-toolchain.toml
       - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
           toolchain: nightly-2023-03-29
           components: rustfmt, clippy
           target: aarch64-unknown-linux-gnu,x86_64-unknown-linux-gnu
+      - uses: goto-bus-stop/setup-zig@v2
+      - run: CARGO_NET_GIT_FETCH_WITH_CLI=true python3 -m pip install cargo-zigbuild
```

Comment:
I would add this as a global env for all jobs? (I think you can do it in the root hierarchy?)